src/main/groovy/wooga/gradle/build/unity/tasks/AbstractUnityBuildEngineTask.groovy

```diff
@@ -0,0 +1,161 @@
+package wooga.gradle.build.unity.tasks
+
+
+import org.gradle.api.file.Directory
+import org.gradle.api.file.RegularFile
+import org.gradle.api.file.RegularFileProperty
+import org.gradle.api.provider.ListProperty
+import org.gradle.api.provider.Property
+import org.gradle.api.provider.Provider
+import org.gradle.api.tasks.Input
+import org.gradle.api.tasks.InputFile
+import org.gradle.api.tasks.Optional
+import wooga.gradle.build.unity.internal.BuildEngineArgs
+import wooga.gradle.secrets.internal.Secrets
+import wooga.gradle.unity.UnityTask
+
+import javax.crypto.spec.SecretKeySpec
+
+
+abstract class AbstractUnityBuildEngineTask extends UnityTask {

```

Comment:
Could you add a logPath property," --logPath", and set it by convention to use the _unityLogFile_ property if that one is present? 

I would like the logs written by UBS to be written to the same location as the unity logs.

Reply:
Should this be a "logPath" meaining a directory in which store logs, or a "logFile" pointing to a file?

Reply:
A logPath since it is to be a directory where to store logs (the engine will output more than 1 file)